src/workerd/server/server.c++

```diff
@@ -5012,7 +5034,7 @@ void Server::startServices(jsg::V8System& v8System,
   // Second pass: Build services.
   for (auto serviceConf: config.getServices()) {
     kj::StringPtr name = serviceConf.getName();
-    auto service = makeService(serviceConf, headerTableBuilder, config.getExtensions());
+    auto service = co_await makeService(serviceConf, headerTableBuilder, config.getExtensions());
```

Comment:
What if there are multiple python workers in the config. Should we be fetching the artifacts concurrently for faster startup? If they rely on the same artifacts, will we end up fetching them multiple times or is there a cache somewhere?

Reply:
There is a cache (PyodideBundleManager and PyodidePackageManager), so the same artifacts won't be fetched multiple times.

I looked into implementing concurrent fetches here, it looks like we have this implemented in EW already (using a CrossThreadWaitList). I think it would make sense to move that list to workerd and reuse it here, but I'd like to do so in a separate PR as it will require quite a bit more refactoring (I think merging the two manager classes may make sense too and I'd like to do it at the same time).